storage/iscsi-tools/iscsid-wrapper/main.go

```diff
@@ -23,18 +23,19 @@ func main() {
 
 	cmd.Stdout = &cmdOut
 
-	if _, err := os.Stat("/etc/iscsi/initiatorname.iscsi"); err != nil {
```

Comment:
how would this work on upgrades? if there already was a name provided?

Reply:
yeh, this would be a change, I've added a release note on Talos side

Reply:
how did it write this file before? where did it go (given that `/etc` is ro)?

Reply:
it was a mount from `/system` but it never worked since `'system` is tmpfs

Reply:
ah, so it was re-created each time...